docs/variants/msi_z690/installation-manual.md

```diff
@@ -56,6 +56,42 @@ backup or alternatively upload it to some cloud or network drive (Ubuntu live
 has a Firefox browser installed). Ubuntu live image is volatile and has no
 persistent storage. All changes made in live image will be lost after reboot.
 
+### Migrating SMBIOS unique data
+
+To migrate the SMBIOS system UUID and board serial number follow the Linux
+instructions below before attempting to flash the binary. The procedure is
+supported on Dasharo version v1.0.0 and later and requires cbfstool built from
+coreboot tree.
+
+```bash
+echo -n `sudo dmidecode -s system-uuid` > system_uuid.txt
+echo -n `sudo dmidecode -s baseboard-serial-number` > serial_number.txt
+# assuming in coreboot root directory
+./build/cbfstool build/coreboot.rom add -f serial_number.txt -n serial_number -t raw -r FW_MAIN_A
+./build/cbfstool build/coreboot.rom add -f serial_number.txt -n serial_number -t raw -r FW_MAIN_B
+./build/cbfstool build/coreboot.rom add -f serial_number.txt -n serial_number -t raw -r COREBOOT
+./build/cbfstool build/coreboot.rom add -f system_uuid.txt -n system_uuid -t raw -r FW_MAIN_A
+./build/cbfstool build/coreboot.rom add -f system_uuid.txt -n system_uuid -t raw -r FW_MAIN_B
+./build/cbfstool build/coreboot.rom add -f system_uuid.txt -n system_uuid -t raw -r COREBOOT
```

Comment:
@miczyg1 @macpijan why that can't be done as one program/script under DTS? IMHO we need kind of https://en.wikibooks.org/wiki/Bash_Shell_Scripting/Whiptail application for quick applications of features, as well as login and other stuff that DTS should provide.

Reply:
Yes it can, but my task included preparing the procedure, so here it is. It is not my responsibility (yet) to move it to DTS as a script.